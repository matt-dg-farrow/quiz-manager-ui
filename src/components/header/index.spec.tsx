import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import React from "react";
import Header from ".";

describe("The Header component", () => {
  const wrapper = shallow(<Header />);

  it("should contain the correct number of components", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
