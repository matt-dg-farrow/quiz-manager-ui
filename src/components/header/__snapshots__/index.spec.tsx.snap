// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Header component should contain the correct number of components 1`] = `
<div
  className="quiz-manager-header-container"
>
  <header
    className="quiz-manager-header"
    data-module="quiz-manager-header"
    role="banner"
  >
    <div
      className="quiz-manager-header-content-container"
    >
      <a
        aria-current="page"
        className="quiz-manager-header-link quiz-manager-header-link--homepage"
        href="/"
      >
        Quiz Manager
      </a>
    </div>
  </header>
</div>
`;
